src/Qbead.h

```diff
@@ -150,20 +257,20 @@ class Qbead {
   const uint8_t ix, iy, iz;
   const bool sx, sy, sz;
   float rbuffer[3];
-  float x, y, z, rx, ry, rz; // filtered and raw acc, in units of g
-  float t_acc, p_acc;        // theta and phi according to gravity
+  float rx, ry, rz;         // filtered and raw acc, in units of g
   float T_imu;             // last update from the IMU
 
-  float t_ble, p_ble; // theta and phi
+  State state;
+
   uint32_t c_ble; // color
 
   static void ble_callback_color(uint16_t conn_hdl, BLECharacteristic* chr, uint8_t* data, uint16_t len) {
       singletoninstance->c_ble =  (data[2] << 16) | (data[1] << 8) | data[0];
   }
 
-  static void ble_callback_theta_phi(uint16_t conn_hdl, BLECharacteristic* chr, uint8_t* data, uint16_t len){
-      singletoninstance->t_ble = data[0]*180/255;
-      singletoninstance->p_ble = data[1]*360/255;
```

Comment:
these should not touch `state` without the user saying so

Reply:
i.e. just reinstate `t_ble` (and potentially rename it to fit your new naming conventions) and have the user decide whether `t_ble` will be used to set `state`